routes.js

```diff
@@ -1,5 +1,6 @@
 const routes = [
     {path: '/', name: 'Home', component: homeComponent},
+    {path: '/admin/addProduct', name: 'Add Product', component: addProductComponent},
```

Comment:
😻 you added route as in alpabatical order 👏🏻

Reply:
Thank You! 🥇